src/instructlab/cli/model/evaluate.py

```diff
@@ -135,6 +137,43 @@
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "--input-questions",
+    type=click.Path(exists=True, path_type=pathlib.Path),
+    default=None,
+    help="Path to the questions and reference answers the model will be evaluating in the DK-Bench evaluation",
+)
+@click.option(
+    "--output-file-formats",
+    type=click.STRING,
+    default="jsonl",
+    show_default=True,
+    help="Comma-separated list of file formats for results of the DK-Bench evaluation. Valid options are csv, jsonl, and xlsx. If this option is not provided the results are written as a .jsonl file",
+)
+@click.option(
+    "--model-prompt",
+    type=click.STRING,
+    default=ragas_eval._DEFAULT_SYSTEM_PROMPT,
+    help="Prompt for the model when getting responses in the DK-Bench evaluation",
+)
+@click.option(
+    "--temperature",
+    type=click.FLOAT,
+    default=0.0,
+    help="Temperature for the model when getting responses in the DK-Bench evaluation",
+)
+@click.option(
+    "--dk-bench-model",
+    type=click.STRING,
+    default=None,
+    help="Model to evaluate for the DK-Bench evaluation",
+)
+@click.option(
+    "--judge-model-name",
```

Comment:
How is this different from the `--judge-model` flag?

Reply:
`--judge-model` is for the path to the judge model for MT-Bench and MT-Bench-Branch evaluation. I can see how it could be confusing having such a similarly named flag. @RobotSail What would you think about removing the ability for an `ilab` user to set the judge model entirely?

Reply:
@alimaredia Ah, I see that makes sense. 

In this case I think we need to redesign the config for evaluations, so that we can set different judge models according to the evaluation we're performing.

This way MT-Bench and its variations can remain set to Prometheus, and DK-Bench here would have a separate config for its own judge model. 


I think this case is actually opposite to the case above with the model we're judging - since the common pattern is we evaluate a single model across different benchmarks, but each benchmark doesn't typically change which judge model it uses. 

Does this make sense?

Reply:
I understand the need to redesign the config and this PR highlighting the need for that. Let's discuss that in follow up work outside of this PR. 

I left the ability to set the judge model in for now. If you think it should be removed I'd be happy to remove it.

Reply:
@alimaredia I agree with that assessment, let's go with your plan:

> Let's discuss that in follow up work outside of this PR.
> 
> I left the ability to set the judge model in for now.